kiwi/schema/kiwi.rnc

```diff
@@ -65,7 +65,7 @@ div {
         attribute xsi:schemaLocation { xsd:anyURI }
     k.image.schemaversion.attribute =
         ## The allowed Schema version (fixed value)
-        attribute schemaversion { "7.6" }
+        attribute schemaversion { "7.7" }
```

Comment:
We only change the schema version on incompatible schema changes which requires an XSLT transformation.  You are only adding stuff which is not an incompatible change. For tracking features according to the version of kiwi we use the main kiwi version not the schema version. Please keep the version as it is

This also obsoletes your XSLT transformation which does basically nothing than changing the version. So please drop the XSLT

Also please drop the adaptions to the various XML snippets that only changes the schema version.
It will make the pull request a lot smaller :)

Thanks much